lib/linguist/languages.yml

```
@@ -550,6 +550,14 @@ Bicep:
   tm_scope: source.bicep
   ace_mode: text
   language_id: 321200902
+Bikeshed:
+  type: programming
+  color: "#5562ac"
+  extensions:
+  - ".bs"
+  tm_scope: source.csswg
+  ace_mode: text
```

This grammar is sufficiently similar enough to HTML that we can repurpose [its CodeMirror and ACE editing modes](https://github.com/github/linguist/blob/7503f7588ca2590cebc4a05af7d00a7e94c0d779/lib/linguist/languages.yml#L2415-L2417):

```suggestion
  ace_mode: html
  codemirror_mode: htmlmixed
  codemirror_mime_type: text/html
```